tomviz/python/GaussianFilterTiltSeries.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name" : "GaussianFilter",
+  "label" : "Gaussian Filter",
```

Comment:
What are the purposes of "name" and "label"? 
I don't quite understand how they control the UI.